build.sbt

```diff
@@ -19,42 +19,41 @@
 // 
 //      
 
-val scalaLibVersion = "2.12.12"
+val scalaLibVersion = "2.13.6"
 val model3Version = "2.7.1-20210602_1321_master"
+val appVersion = a8.sbt_a8.versionStamp(file("."))
+
 
 scalacOptions in Global ++= Seq("-deprecation", "-unchecked", "-feature")
 
-resolvers in Global += "a8-repo" at Common.readRepoUrl()
+//resolvers in Global += "a8-repo" at Common.readRepoUrl()
 
-publishTo in Global := Some("a8-repo-releases" at Common.readRepoUrl())
+//publishTo in Global := sonatypePublishToBundle.value
```

Comment:
This will need to be uncommented to publish.

Reply:
Yes I plan to use the updated "public" option in versions on itself to get this all properly setup